operator/v1/types_console.go

```diff
@@ -106,6 +106,50 @@ type ConsoleCustomization struct {
 	// SVG format preferred
 	// +optional
 	CustomLogoFile configv1.ConfigMapFileReference `json:"customLogoFile,omitempty"`
+	// developerCatalog allows to configure the shown developer catalog categories.
+	// +kubebuilder:validation:Optional
+	// +optional
+	DeveloperCatalog DeveloperConsoleCatalogCustomization `json:"developerCatalog,omitempty"`
+}
+
+// DeveloperConsoleCatalogCustomization allow cluster admin to configure developer catalog.
```

Comment:
I feel like this is an unnecessary intermediate property. Everything under the `developerCatalog` the is technically customization.

Reply:
But this does not affect operator hub. So this makes it more clear at the moment? But at the end, its your decision.

Reply:
Sorry I misread the API. I thought there was a customization stanza under `developerCatalog`. Please disregard.